ansible/roles/plpsql/templates/recovery.conf

```diff
@@ -1,4 +1,4 @@
 standby_mode = 'on'
 primary_slot_name = 'amsmetadb_3'
-primary_conninfo = 'host=hkgmetadb.infra.ooni.io port=5432 user=amsrepl password={{ amsrepl_hkgmetadb_password }}'
+primary_conninfo = 'host=hkgmetadb.infra.ooni.io port=5432 user=amsrepl sslcert=/srv/pl-psql/postgres_amsrepl_user.cert sslcert=/srv/pl-psql/postgres_amsrepl_user.key'
```

Comment:
Is `.vault` suffix auto-stripped?

Reply:
My understanding is that the copy command will transparently decrypt a vault file and copy it unencrypted (see: https://github.com/ansible/ansible/pull/15417).

Reply:
It will decrypt, yes. Will it strip `.vault` suffix from the filename? :)

Reply:
Hum I see what you mean. Yeah I suspect you are right that it will have the vault prefix because the `dest` argument has the item name with the `vault` suffix. I will look into fixing that.